lib/rules/require-meta-schema.js

```diff
@@ -104,7 +104,16 @@ module.exports = {
           hasEmptySchema = true;
         }
 
-        if (!['ArrayExpression', 'ObjectExpression'].includes(value.type)) {
+        if (
+          ![
+            'ArrayExpression',
+            'CallExpression',
+            'ConditionalExpression',
+            'LogicalExpression',
+            'MemberExpression',
+            'ObjectExpression',
+          ].includes(value.type)
+        ) {
```

Comment:
just wondering we can explicitly say what we want to check - as more expressions could be added in future es.

e.g some new added expressions:
* ChainExpression
* LogicalAssignmentExpression

Reply:
Yeah I'm worried about this too. I think I should update this to instead only check a few common invalid types like `Literal`.

Reply:
Switched to only checking for literals / null / undefined to be safe.